chains/eip155-10.json

```diff
@@ -17,21 +17,21 @@
     "decimals": 18
   },
   "infoURL": "https://optimism.io",
-  "shortName": "oeth",
+  "shortName": "optimism",
```

Comment:
Are you sure this is the canonical shortname for optimism? e.g. safe url prefix is oeth

Reply:
I am not sure. To reduce the scope I only made updates to existing chains that were missing. Here is the original [commit](https://github.com/polymerdao/polymer-registry/commit/6b6d75d3217e02049c6b274e129acabf58c70a2b) that changed the short name